src/pages/editor/RightsRetentionStrategySettings.tsx

```diff
@@ -30,10 +30,10 @@ export const RightsRetentionStrategySettings = () => {
   const updateRightsRetentionStrategy = useMutation({
     mutationFn: (customer: CustomerInstitution) => updateCustomerInstitution(customer),
     onSuccess: (response) => {
+      dispatch(setCustomer(response.data));
       dispatch(
         setNotification({ message: t('feedback.success.update_rights_retention_strategy'), variant: 'success' })
       );
-      dispatch(setCustomer(response.data));
```

Comment:
Could not find bug in this component, but I changed the order of these calls. Only show the success message if it also were able to update redux.